test/Lazy/takeRight.spec.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  delay,
+  filter,
+  map,
+  pipe,
+  range,
+  takeRight,
+  toArray,
+  toAsync,
+  concurrent,
+} from "../../src/index";
+import { Concurrent } from "../../src/Lazy/concurrent";
+import { callFuncAfterTime, generatorMock } from "../utils";
+
+describe("take", function () {
```

Comment:
```suggestion
describe("takeRight", function () {
```

Reply:
thanks for your suggestion and I changed this and below one!